test/integration/noderesourcetopology_test.go

```diff
@@ -795,3 +1039,87 @@ func makeNodeResourceTopology(name string, tmp topologyv1alpha1.TopologyManagerP
 	}
 	return nrt
 }
+
+func parseTestUserEntry(entries []nrtTestUserEntry, ns string) []nrtTestEntry {
+	var teList []nrtTestEntry
+	for i, e := range entries {
+		p := st.MakePod().Name(fmt.Sprintf("%s-%d", "topology-aware-scheduler-pod", i+1)).Namespace(ns)
+		for _, req := range e.initCntReq {
+			p = initWithReqAndLimit(p, req)
+		}
+
+		for _, req := range e.cntReq {
+			p = withReqAndLimit(p, req)
+		}
+		nodeTopologies := []*topologyv1alpha1.NodeResourceTopology{
+			{
+				ObjectMeta:       metav1.ObjectMeta{Name: "fake-node-1"},
+				TopologyPolicies: []string{string(topologyv1alpha1.SingleNUMANodeContainerLevel)},
+				Zones: topologyv1alpha1.ZoneList{
+					{
+						Name: "node-0",
+						Type: "Node",
+						Resources: topologyv1alpha1.ResourceInfoList{
+							noderesourcetopology.MakeTopologyResInfo(cpu, "32", "30"),
+							noderesourcetopology.MakeTopologyResInfo(memory, "64Gi", "60Gi"),
+							noderesourcetopology.MakeTopologyResInfo(hugepages2Mi, "384Mi", "384Mi"),
+							noderesourcetopology.MakeTopologyResInfo(nicResourceName, "16", "16"),
+						},
+					},
+					{
+						Name: "node-1",
+						Type: "Node",
+						Resources: topologyv1alpha1.ResourceInfoList{
+							noderesourcetopology.MakeTopologyResInfo(cpu, "32", "32"),
+							noderesourcetopology.MakeTopologyResInfo(memory, "64Gi", "64Gi"),
+							noderesourcetopology.MakeTopologyResInfo(hugepages2Mi, "512Mi", "512Mi"),
+							noderesourcetopology.MakeTopologyResInfo(nicResourceName, "32", "32"),
+						},
+					},
+				},
+			},
+			{
+				// we set all available resource as 0
+				// because we want all these tests to be running against fake-node-1
```

Comment:
```NodeResourceFit``` won't filter ```fake-node-2``` because it has enough resources. 
Next, if we won't create ```noderesourcetopology``` object for ```fake-node-2``` with zero resources, it won't be filtered by our plugin either.
IOW the node would be marked as a suitable one and it will affect the tests results`

Reply:
Yeah, I recall now a scenario that was pointed out to us earlier where NRT objects did not exist. Thanks for explaining.